.github/workflows/chores.yml

```
@@ -87,6 +90,21 @@ jobs:
+          doxygen /tmp/Doxyfile
+          mv html /tmp
+          rm -rf latex
+          git checkout gh_pages
+          git rm -rf .
+          mv /tmp/html/* .
+          git add .
+          git commit --amend --no-edit
+      - name: Pushing documentation
+        env:
+          PAT: ${{ secrets.PAT }}
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:39: Double quote to prevent globbing and word splitting

(shellcheck)